services/ext/service.go

```diff
@@ -533,7 +534,30 @@ func (s *Service) GetCommunityID(tokenURI string) string {
 	return ""
 }
 
-func (s *Service) FillCollectibleMetadata(collectible *thirdparty.FullCollectibleData) error {
+func (s *Service) FillCollectiblesMetadata(communityID string, cs []*thirdparty.FullCollectibleData) (bool, error) {
+	if s.messenger == nil {
+		return false, fmt.Errorf("messenger not ready")
+	}
+
+	community, err := s.fetchCommunityInfoForCollectibles(communityID, collectibles.IDsFromAssets(cs))
+	if err != nil {
+		return false, err
+	}
+	if community == nil {
+		return false, nil
+	}
+
+	for _, collectible := range cs {
+		err := s.FillCollectibleMetadata(community, collectible)
+		if err != nil {
+			return true, err
+		}
+	}
+
+	return true, nil
+}
+
+func (s *Service) FillCollectibleMetadata(community *communities.Community, collectible *thirdparty.FullCollectibleData) error {
```

Comment:
missed at the other PR, but this probably doesn't need to be public anymore

Reply:
True, that's a detail, will leave it as is for now if you don't mind.